ui/arduino/views/components/repl-panel.js

```diff
@@ -50,7 +50,7 @@ function ReplOperations(state, emit) {
     Button({
       icon: 'delete.svg',
       size: 'small',
-      tooltip: 'Clean',
+      tooltip: `Clean (${state.platform === 'darwin' ? 'Cmd' : 'Ctrl'}+k)`,
```

Comment:
Since the REPL is a terminal like view, I'd go with the well known ctrl + L shortcut
```suggestion
      tooltip: `Clean (${state.platform === 'darwin' ? 'Cmd' : 'Ctrl'}+L)`,
```